src/gt4py/ir/nodes.py

```diff
@@ -287,9 +287,10 @@ def __str__(self):
 
 
 @enum.unique
-class AccessIntent(enum.Enum):
-    READ_ONLY = 0
-    READ_WRITE = 1
+class AccessIntent(enum.IntFlag):
+    READ = 1
+    WRITE = 2
+    READ_WRITE = 3
```

Comment:
Isn't `READ_WRITE` redundant here? I mean, if `AccessIntent` is a `Flag`, it should be possible to set both `READ` and `WRITE` at the same time, or did I get it wrong?

Reply:
It's so that it can be specified as a shortcut for `AccessIntent.READ & AccessIntent.WRITE`.

Reply:
We could instead use `READ_WRITE = READ | WRITE`